.github/workflows/apt.yml

```diff
@@ -46,7 +46,17 @@ jobs:
           sudo apt -V install ruby ruby-bundler ruby-serverspec
           sudo gem install bundler:2.2.9 --no-document
           sudo gem uninstall fileutils
+      - name: cache deb
+        uses: actions/cache@v4
+        id: cache-deb
+        with:
+          path: |
+            fluent-package/apt/repositories
+            fluent-apt-source/apt/repositories
+            fluent-lts-apt-source/apt/repositories
+          key: ${{ runner.os }}-cache-${{ matrix.rake-job }}-${{ hashFiles('**/config.rb', '**/Rakefile', '**/Gemfile*', '**/deb/*') }}
```

Comment:
Using `hashFiles` is an excellent idea.
but is this enough? Don't we need to include more files?

* `fluent-package/templates/**`
* `fluent-package/debian/**`
* files under `fluent-package/apt/` except for test files, such as Dockerfile

Reply:
Thanks, take into account more files .